docs/linux_dependencies.md

```diff
@@ -23,8 +23,40 @@ Compiling with clang is also possible - replace the `g++` package with `clang`.
 
 ### Windows Subsystem for Linux (WSL 2)
 
-Graphics and audio need to be configured for them to work with WSL 2 backend.
-Please see the ubuntu [WSL documentation](https://wiki.ubuntu.com/WSL) on how to set up graphics and audio.
+WSL2's virtual GPU Driver only supports OpenGL, requiring you to cross-compile for Windows within WSL and then run the build on Windows. (See [#841](https://github.com/bevyengine/bevy/issues/841) and [#7790](https://github.com/microsoft/WSL/issues/7790))
```

Comment:
Please use the 'full syntax' for linking to non-bevyengine/bevy issues (i.e. `[microsoft/WSL#7790]`)